RELEASE-NOTES.txt

```diff
@@ -21,7 +21,7 @@
 * [**] Block Editor: Added Contact Info block to sites on WPcom or with Jetpack version >= 8.5.
 * [**] We updated the app's color scheme with a brighter new blue used throughout. [#16213, #16207]
 * [**] We updated the login prologue with brand new content and graphics. [#16159, #16177, #16185, #16187, #16200, #16217, #16219, #16221, #16222]
-* [**] We updated the app's color scheme with a brighter new blue used throughout. [#16213, #16207] 
+* [**] We updated the app's color scheme with a brighter new blue used throughout. [#16213, #16207]
```

Comment:
There's a couple of unexpected changes here to the release notes, are these meant to be here?

Reply:
Ah good call this is a result of the text editor I use cleaning up trailing whitespace characters. Not intentional I just missed excluding this line.

Reply:
Gotcha, I hadn't noticed that it was just a whitespace change.